detect_secrets/core/usage.py

```diff
@@ -51,36 +51,37 @@ def add_output_verified_false_flag(parser):
 
 
 class ParserBuilder(object):
+
     def __init__(self):
         self.parser = argparse.ArgumentParser()
-        self.subparser = None
 
         self.add_default_arguments()
 
     def add_default_arguments(self):
-        self._add_verbosity_argument()._add_version_argument()
+        self._add_verbosity_argument()\
+            ._add_version_argument()
 
     def add_pre_commit_arguments(self):
         self._add_filenames_argument()\
             ._add_set_baseline_argument()\
             ._add_exclude_lines_argument()\
             ._add_word_list_argument()\
             ._add_use_all_plugins_argument()\
-            ._add_no_verify_flag()\
+            ._add_no_verify_flag() \
             ._add_output_verified_false_flag()\
-            ._add_fail_on_unaudited_flag()
+            ._add_fail_on_non_audited_flag()
```

Comment:
This line should be `._add_fail_on_unaudited_flag()`: https://github.com/IBM/detect-secrets/blob/master/detect_secrets/core/usage.py#L88